compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -69,13 +69,14 @@ class EventNotifierImpl final : public EventNotifier
 
 } // namespace
 
-Interpreter::Interpreter(const luci::Module *module)
+Interpreter::Interpreter(const luci::Module *module, luci_interpreter::MManager *memory_manager)
```

Comment:
There are more tools than EvalDriver that use interpreter, so in case `memory_manager == nullptr` they will break. I propose to:
- introduce new constructor with MManager
- from old contructor: create SimpleMemory Manager and call new one

This "temporary" solution, to make tests pass =)
I hope we will remove this after all components using interpreter will adopt new interface.